tests/_async/test_district_case.py

```diff
@@ -17,6 +17,7 @@ class TestGetDistrictCase:
     async def test_get_district_case_integer_fuzz(self, cases):
 
         response = await self.client.get_district_cases(cases=cases)
+        print(response)
         assert response.get("detail") == "case_id not found"
```

Comment:
Do we want this print statement in here?

Reply:
fixed